src/lofreq/call/config.vsh.yaml

```
@@ -0,0 +1,249 @@
+
+    LoFreq* can run on almost any type of aligned sequencing data (e.g. Illumina, IonTorrent or Pacbio) since no machine- or sequencing-technology dependent thresholds are used. It automatically adapts to changes in coverage and sequencing quality and can therefore be applied to a variety of data-sets e.g. viral/quasispecies, bacterial, metagenomics or somatic data.
+
+    LoFreq* is very sensitive; most notably, it is able to predict variants below the average base-call quality (i.e. sequencing error rate). Each variant call is assigned a p-value which allows for rigorous false positive control. Even though it uses no approximations or heuristics, it is very efficient due to several runtime optimizations and also provides a (pseudo-)parallel implementation. LoFreq* is generic and fast enough to be applied to high-coverage data and large genomes. On a single processor it takes a minute to analyze Dengue genome sequencing data with nearly 4000X coverage, roughly one hour to call SNVs on a 600X coverage E.coli genome and also roughly an hour to run on a 100X coverage human exome dataset.
+  info:
+    keywords: [ "variant calling", "low frequancy variant calling", "lofreq", "lofreq/call"]
+    homepage: https://csb5.github.io/lofreq/
+    documentation: https://csb5.github.io/lofreq/commands/
+    reference: doi:10.1093/nar/gks918
+    license: "MIT"
+  requirements:
+    commands: [ lofreq ]
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: --input
+          type: file
+          description: |
+            Input BAM file.
+          required: true
+          example: "normal.bam"
+        - name: --input_bai
+          type: file
+          description: |
+            Index file for the input BAM file.
+          required: true
+          example: "normal.bai"
+        - name: --ref
+          alternatives: -f
+          type: file
+          description: |
+            Indexed reference fasta file (gzip supported) [null].
+          required: true
+          example: "reference.fasta"
+    - name: Outputs
+      arguments:
+        - name: --output
+          alternatives: -o
+          type: file
+          description: |
+            Vcf output file [- = stdout].
+          required: true
+          direction: output
+          example: "output.vcf"
+    - name: Arguments
+      arguments:
+        - name: --region
+          alternatives: -r
+          type: string
+          description: |
+            Limit calls to this region (chrom:start-end) [null].
+          required: false
+          example: "chr1:1000-2000"
+        - name: --bed
+          alternatives: -l
+          type: file
+          description: |
+            List of positions (chr pos) or regions (BED) [null].
+          required: false
+          example: "regions.bed"
+        - name: --min_bq
+          alternatives: -q
+          type: integer
+          description: |
+            Skip any base with baseQ smaller than INT [6].
```

Make the default values a little more explicit:

```suggestion
            Skip any base with baseQ smaller than INT. Default: 6.